pwndbg/wrappers/readelf.py

```diff
@@ -33,3 +34,49 @@ def _extract_jumps(line):
             return False
     except IndexError:
         return False
+
+@pwndbg.wrappers.OnlyWithCommand(cmd_name)
+def get_load_segment_info():
+    '''
+    Looks for LOAD sections by parsing the output of `readelf -l <binary>`
+    '''
+    local_path = pwndbg.file.get_file(pwndbg.proc.exe)
+    cmd = [get_jmpslots.cmd_path, "-l", local_path]
+    readelf_out = pwndbg.wrappers.call_cmd(cmd)
+
+    segments = []
+    load_found = False
+
+    # Output from readelf is 
+    # Type           Offset             VirtAddr           PhysAddr
+    #                FileSiz            MemSiz             Flags  Align
+    # LOAD           0x0000000000000000 0x0000000000000000 0x0000000000000000
+    #                0x0000000000000830 0x0000000000000830  R E    0x200000
+    # Account for this using two regular expressions
+    re_first = r"\s+LOAD\s+(0x[0-9A-Fa-f]+) (0x[0-9A-Fa-f]+) (0x[0-9A-Fa-f]+)"
+    re_secnd = r"\s+(0x[0-9A-Fa-f]+) (0x[0-9A-Fa-f]+)  (.)(.)(.)\s+(0x[0-9A-Fa-f]+)"
```

Comment:
Hm... lets compile those regexes before function definition and since they are constants it might be good to use uppercase naming for them